nbextensions/usability/collapsible_headings/main.js

```diff
@@ -592,11 +592,12 @@ define([
 		new_cell.set_heading_level(level);
 		new_cell.code_mirror.setSelection({line:0, ch: level + 1}, {line:0, ch: level + 1 + new_text.length});
 		Jupyter.notebook.select(index, true);
+		Jupyter.notebook.execute_cell(); // to make sure the new heading is "registered" by the plugin
```

Comment:
Didn't realize the implications of the `execute_cell()`. Pushed up a version with `select_reveals` moved instead. Thanks!